### ISSUE TITLE
CPU版本报错

### ISSUE THREAD

**Issue by kingmacth** [YaoFANGUK/video-subtitle-extractor#262]
D:\Anaconda3\envs\videoEnv\lib\site-packages\setuptools\sandbox.py:13: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  import pkg_resources
D:\Anaconda3\envs\videoEnv\lib\site-packages\pkg_resources\__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('mpl_toolkits')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)
D:\Anaconda3\envs\videoEnv\lib\site-packages\pkg_resources\__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('google')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)

CPU版本按照说明安装，运行python gui.py时报这个错误

**Comment by YaoFANGUK** [comment]
你告诉我哪一条是报错

**Comment by kingmacth** [comment]
> 你告诉我哪一条是报错

对。你这么说也有道理。不是报错。但是我运行它chrome会崩溃。提示Out of Memory。程序就结束运行了。也不会出ui界面。

**Comment by eritpchy** [comment]
看看内存占用
